plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js

```diff
@@ -22,12 +22,6 @@ const helpLinks = [
     text: 'Integration with external APIs',
     href: webDocumentationLink('user-manual/manager/manual-integration.html'),
   },
-  {
-    text: 'VirusTotal integration',
-    href: webDocumentationLink(
-      'user-manual/capabilities/malware-detection/virus-total-integration.html',
-    ),
-  },
```

Comment:
**nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.